payjoin-cli/src/app/v2.rs

```diff
@@ -130,27 +139,20 @@ impl AppTrait for App {
 }
 
 impl App {
-    fn construct_payjoin_uri(
+    async fn construct_payjoin_uri(
```

Comment:
why this need to be async?

Reply:
Whoops nice catch. It doesn't.

I actually think the `Enrolled` struct has all of the information necessary to construct a Uri and should become available from one of its methods.